.github/workflows/push.yml

```
@@ -1,35 +1,15 @@
 name: CI/CD
-on: push
-jobs:
-  setup:
-    name: setup
-    runs-on: ubuntu-latest
-    steps:
-      - name: Dump GitHub context
-        env:
-          GITHUB_CONTEXT: ${{ toJson(github) }}
-        run: echo "$GITHUB_CONTEXT"
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Setup
-        uses: actions/setup-node@v3
-        with:
-          node-version: 18
-      - uses: actions/cache@v1
-        id: yarn-cache
-        with:
-          path: node_modules
-          key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-node_modules-
-      - name: Install
-        if: steps.yarn-cache.outputs.cache-hit != 'true'
-        run: yarn install
+on:
+  push:
+    branches:
+    - main
+    - add-ci-cd
+  pull_request_target:
 
+jobs:
   test:
     name: Test
     runs-on: ubuntu-latest
-    needs: setup
     services:
       mongo:
         image: mongo:3.4.23
```

This version of `mongo` is *old*

```suggestion
        image: mongo:5.0
```